<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainFrame</class>
 <widget class="QWidget" name="MainFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>619</width>
    <height>317</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(194, 190, 171);</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="NavButton" name="homeButton">
        <property name="text">
         <string>首页</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="NavButton" name="attendButton">
        <property name="text">
         <string>出勤</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="NavButton" name="estateButton">
        <property name="text">
         <string>房产</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="NavButton" name="repairButton">
        <property name="text">
         <string>故障报修</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="NavButton" name="personnelButton">
        <property name="text">
         <string>人员管理</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="NavButton" name="ownerButton">
        <property name="text">
         <string>业主管理</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="NavButton" name="carButton">
        <property name="text">
         <string>车位管理</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="NavButton" name="payButton">
        <property name="text">
         <string>缴费管理</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="NavButton" name="settingsButton">
        <property name="text">
         <string>系统设置</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>111</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="NavButton" name="exitButton">
        <property name="text">
         <string>退出系统</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>NavButton</class>
   <extends>QPushButton</extends>
   <header>CustomWidget/navbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
